services/version-api/helpers/getVersionConfigurations.js

```diff
@@ -0,0 +1,21 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import params from '../../../libs/params';
+import config from '../../../config';
+
+/**
+ * Fetch application version configurations from AWS SSM Parameter store
+ *
+ * @returns {Promise<{versions: {ios: {min: string, max: string}, android: {min: string, max: string}}}>} SSM parameters
+ */
+async function getVersionConfigurations() {
+  const [error, response] = await to(params.read(config.version.envsKeyName));
+  if (error) {
+    throwError(500);
```

Comment:
This helper (throwError) should only be used in the main lambda function. Instead you could use the javascript 'throw' together with an relevant error message for this context.

Reply:
Removed entirely

Reply:
re-added. The function throwError fits very well in this case since it throws a generic error message with a status code of 500. The lambda function itself should not throw but instead it should return a http response